### ISSUE TITLE
home page get Grayed

### ISSUE THREAD

**Issue by Bleustacks92** [tahaak67/Farhan#54]
when i exit from an application (in the white list) to an other application, this one gets grayed.
For exemple when i exit to home it get grayed.

**Comment by trixter13** [comment]
Same issue here - I tested the grayscale option but now everything on my device is grayscale and I can't figure out how to turn it off. Turning off grayscale and the global switch in Farhan don't do anything

**Comment by tahaak67** [comment]
@trixter13 
turning off the switch in grayscale settings screen should revert it, don't use the global switch
btw what devices are you guys using?

**Comment by trixter13** [comment]
> @trixter13 turning off the switch in grayscale settings screen should revert it, don't use the global switch
I tried that under Settings > Accessibility > Color and motion > Color correction but the toggle to use color correction was not enabled so that didn't work either. I had to reboot my device.

I'm using a Pixel running GrapheneOS.

**Comment by tahaak67** [comment]
ah, i was refereeing to the grayscale option in Farhan when you tap "Grayscale" it should take you to a screen with a switch on top if you tap it it should turn off and disable any active grayscaling

**Comment by trixter13** [comment]
> ah, i was refereeing to the grayscale option in Farhan itself when you tap "Grayscale" it should take you to a screen with a switch on top if you tap it it should turn off and disable any active grayscaling

Yes that switch also does nothing.

**Comment by ItsIgnacioPortal** [comment]
Yes this is deffinetly an issue with GrapheneOS rather than with farhan. See: https://github.com/GrapheneOS/os-issue-tracker/issues/6001

**Comment by tahaak67** [comment]
This is an issue with pixel devices and a workaround has been implemented in the latest version released yesterday
